framework/src/messaging/Kconfig

```diff
@@ -0,0 +1,28 @@
+#
+# For a description of the syntax of this configuration file,
+# see kconfig-language at https://www.kernel.org/doc/Documentation/kbuild/kconfig-language.txt
+#
+
+config MESSAGING_IPC
+	bool "Enable Messaging APIs"
+	default y
+	depends on !DISABLE_MQUEUE
+	depends on !DISABLE_SIGNALS
+	---help---
+		Enables Messaging APIs.
```

Comment:
Let's add description to say what it is, what does support, when it can be used and etc

Reply:
Done.